src/components/Network.tsx

```diff
@@ -85,36 +84,35 @@ export const NetworkSides = () => {
       <Box p={1} className={classes.statsEntry}>
         <Typography variant="h4">
           <IconApiStatus className={classes.svg} status={targetReady} />
-          <a target="_blank" rel="noreferrer" href={targetChainDetails.targetPolkadotjsUrl}>
-            {targetChainDetails.targetChain}
+          <a target="_blank" rel="noreferrer" href={targetChainDetails.polkadotjsUrl}>
+            {targetChainDetails.chain}
           </a>
         </Typography>
-        <span style={{ opacity: 0.4 }}># {dbTarget.bestBlock}</span>
+        <span style={{ opacity: 0.4 }}># {targetChainValues.bestBlock}</span>
       </Box>
     </Box>
   );
 };
 
 export const NetworkStats = () => {
   const classes = useStyles();
-  const dbSource = useDashboard(ChainDetails.SOURCE);
-  const dbTarget = useDashboard(ChainDetails.TARGET);
+  const { sourceChainValues, targetChainValues } = useOnChainValuesContext();
```

Comment:
useDashboard hook was removed from dependent components and corresponding values are read now from a context.